inductiva/tasks/task.py

```diff
@@ -153,14 +154,14 @@ def set_output_class(self, output_class):
         """Set the output class of the task."""
         self._output_class = output_class
 
-    def get_output(self, output_dir=None):
+    def get_output(self, output_dir: Optional[types.Path] = None):
         """Get the output of the task.
 
         Returns:
             The output of the task.
         """
         self.wait()
-        output_dir = self.download_outputs(output_dir)
+        output_dir = self.download_outputs(output_dir=output_dir)
```

Comment:
this operation can take forever, right?
Do you want to warn the user, or does donwload_outputs already inform the user?

Reply:
The download_outputs already gives this information: <img width="490" alt="image" src="https://github.com/inductiva/inductiva/assets/114397668/1b7529bd-4b7b-48e5-905e-f3d0b16cbf68">

Reply:
Then add this to the comments. You can actually copy paste this info there, so that people know.

Reply:
Done.